src/store/matches/actions.ts

```diff
@@ -1,13 +1,48 @@
-import {Match, GET_MATCHES, MatchActionTypes} from './types'
-import {bindActionCreators, Dispatch, AnyAction} from 'redux';
-import * as mocks from '../../mocks';
-import {ThunkAction} from 'redux-thunk';
+import {Match} from '../../models'
+import {GET_MATCHES, MatchActionTypes} from './types'
+import {error as Error} from '../types'
+import {bindActionCreators, Dispatch, AnyAction, ActionCreator} from 'redux';
+import {GetData} from '../../services';
+import {ThunkAction, ThunkDispatch} from 'redux-thunk';
+import realm from '../../realm';
 
 export const getMatches = (): ThunkAction<MatchActionTypes, {}, {}, AnyAction> => (
   dispatch: Dispatch,
 ): MatchActionTypes => {
+  const matches = realm.objects<Match>("Match").map(realmObject => ({
+    ...realmObject
+  }))
   return dispatch({
     type: GET_MATCHES,
-    matches: mocks.matches,
+    matches,
   });
+};
+
+export const loadMatches : ActionCreator<
```

Comment:
Same thing as events, so just reply to that one.

Reply:
right now I have it so load matches will like sync with the database where get matches will retrieve from the local relam?